content_scripts/ui_component.coffee

```diff
@@ -62,10 +62,11 @@ class UIComponent
                       @hide false
                     false # We will not be calling sendResponse.
                   # If this frame receives the focus, then hide the UI component.
-                  window.addEventListener "focus", (event) =>
+                  window.addEventListener "focus", ((event) =>
                     if event.target == window and @options?.focus
                       @hide false
-                    true # Continue propagating the event.
+                    true
+                  ), true # Continue propagating the event.
```

Comment:
The comment belongs to the `true` on the line above, doesn’t it?

Reply:
updated.

in fact, the first 'true' here does nothing - Browser doesn't use true/false to decide whether to continue on such listeners